compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -514,9 +514,9 @@ data DependencyInfo = DependencyInfo
   -- Note that for data-dependencies it is sufficient to list a DAR without
   -- listing all of its dependencies.
   , mainUnitIds :: [UnitId]
-  -- ^ Unit id of the main DALFs specified in dependencies and
-  -- data-dependencies. This will be used to generate the --package
-  -- flags which define which packages are exposed by default.
+  -- ^ Unit id of the main DALFs specified in dependencies.
```

Comment:
It still applies to DARs in data-dependencies right? I think the comment is mostly fine. What change is the definition of "main dalf" which no longer applies to indvidual dalfs.